core/style.css

```diff
@@ -836,6 +836,23 @@ h3.title {
   font-size: 16px;
 }
 
+/* history details */
+#historydetails {
+  text-align: center;
+  color: #ffce00;
+}
+
+#historydetails .missing {
+}
+
+#historydetails span {
+  color: #ff4a4a;
+}
+
+#historydetails span.done {
```

Comment:
I wonder which classname fit best: done, completed, checked-in, ...

Reply:
I don't know, completed is better than done
we can also use checkbox

Reply:
That's why `checked` (or `checked-in`) makes sense too.